src/components/WhereToDonateCard/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import Heading from '#components//Heading';
+import Link from '#components//Link';
+
+import styles from './styles.module.css';
+
+type WhereToDonateAddress = {
+    title: string;
+    time: string;
+    organizer: string;
+    location?: string;
+};
+
+interface Props {
+    whereToDonateItems: WhereToDonateAddress[];
+}
+
+export default function WhereToDonateCard({ whereToDonateItems }: Props) {
+    return (
+        <div className={styles.whereToDonateDescription}>
+            {whereToDonateItems?.map((item, index) => {
+                const {
+                    title, time, organizer, location,
+                } = item;
+                return (
+                    <div key={`${title}-${time}`} className={styles.whereToDonateItems}>
```

Comment:
Let's separate out props into separate lines.
Don't we have ID that we can use as key?